src/test/java/org/jabref/logic/pdf/PdfAnnotationImporterTest.java

```diff
@@ -0,0 +1,130 @@
+package org.jabref.logic.pdf;
+
+import java.nio.file.Paths;
+import java.util.List;
+
+import org.jabref.model.pdf.FileAnnotation;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class PdfAnnotationImporterTest {
+
+    private AnnotationImporter importer = new PdfAnnotationImporter();
+
+    @Test
+    public void noAnnotationsWriteProtected() {
+
+        List<FileAnnotation> annotations = importer.importAnnotations(Paths.get("src/test/resources/pdfs/write-protected.pdf"));
+        assertEquals(0, annotations.size());
+    }
+
+    @Test
+    public void noAnnotationsEncrypted() {
+        List<FileAnnotation> annotations = importer.importAnnotations(Paths.get("src/test/resources/pdfs/encrypted.pdf"));
+        assertEquals(0, annotations.size());
+    }
+
+    @Test
+    public void twoAnnotationsThesisExample() {
+
+        List<FileAnnotation> annotations = importer.importAnnotations(Paths.get("src/test/resources/pdfs/thesis-example.pdf"));
+        assertEquals(2, annotations.size());
+    }
+
+    @Test
+    public void noAnnotationsMinimal() {
+        List<FileAnnotation> annotations = importer.importAnnotations(Paths.get("src/test/resources/pdfs/minimal.pdf"));
+        assertEquals(0, annotations.size());
+    }
+
+    @Test
+    public void inlineNoteMinimal() {
+        List<FileAnnotation> annotations = importer.importAnnotations(Paths.get("src/test/resources/pdfs/minimal-inlinenote.pdf"));
+        assertEquals(1, annotations.size());
```

Comment:
The junit message would show the objects values for exepcted vs actual as it is done in the fetchers or importers for example.
So you can directly compare them and see any differences, here you would get only Expected 1 actual 0 in case a test would fail -> Not helpful